content/cpp/concepts/math-functions/terms/log2/log2.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: 'log2()'
+Description: 'Returns the base-2 logarithm of the argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`log2()`** function returns the base-2 logarithm of the argument.
+
+## Syntax
+
+```pseudo
+log2(x)
+```
+
+If the `x` parameter is negative (less than zero), then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressly returned.
```

Comment:
@ddiegow I notice a separate entry `log10()` has the same explanation for pole errors. Let's just remove the term "expressly" here because there is a dual meaning which can be confusing (quickly or explicitly).

Reply:
Hi, made the change, created a commit and pushed it.